### ISSUE TITLE
'environment' property not set

### ISSUE THREAD

**Issue by RobKenis** [serenity-bdd/serenity-core#1990]
## Issue
When running serenity with `-Denvironment`, the environment is not set, but remains _null_.
Example command: `./gradlew clean test aggregate -Denvironment=dev`

## Stacktrace
```Environment 'host.url.base' property undefined for environment 'null'
net.serenitybdd.core.environment.UndefinedEnvironmentVariableException: Environment 'host.url.base' property undefined for environment 'null'
	at net.serenitybdd.core.environment.EnvironmentSpecificConfiguration.lambda$getProperty$3(EnvironmentSpecificConfiguration.java:73)
	at net.serenitybdd.core.environment.EnvironmentSpecificConfiguration$$Lambda$112/0000000000000000.get(Unknown Source)
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at net.serenitybdd.core.environment.EnvironmentSpecificConfiguration.getProperty(EnvironmentSpecificConfiguration.java:69)
	at com.nike.emeasc.node.hashing.test.steps.ContextSteps.aRequestToPathReturnsStatusStatus(ContextSteps.java:26)
	at ✽.a request to '/actuator/health' returns status '200'(src/test/resources/features/context/start_context.feature:5)
```

### Breaking setup:
- JVM: 11.0.6 (Eclipse OpenJ9 openj9-0.18.0)
- Gradle: 6.0.1
- serenity-core: 2.0.64
- serenity-cucumber4: 1.0.19
- cucumber-core: 4.2.0
- cucumber-junit: 4.2.0

### Working setup:
- JVM: 11.0.6 (Eclipse OpenJ9 openj9-0.18.0)
- Gradle: 6.0.1
- serenity-core: 2.0.48
- serenity-cucumber4: 1.0.12
- cucumber-core: 4.2.0
- cucumber-junit: 4.2.0

**Comment by wakaleo** [comment]
This is fixed in 2.1.x

**Comment by RobKenis** [comment]
Thank you, it is working with 2.1.5
Have a nice day!
